bundled_scripts/ilo.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+import sys
+from copy import deepcopy
+
+import hpilo
+
+
+USER = os.environ['MANAGEMENT_USER_NAME']
+PASS = os.environ['MANAGEMENT_USER_PASSWORD']
+
+MAC_PREFIX_BLACKLIST = [
+    '505054', '33506F', '009876', '000000', '00000C', '204153', '149120',
+    '020054', 'FEFFFF', '1AF920', '020820', 'DEAD2C', 'FEAD4D',
+]
+
+DEVICE_INFO_TEMPLATE = {
```

Comment:
if it's template for ralph-cli, and since most of our script will be written in python, maybe we could make a simple, tiny python package with base class for the plugin and some utilities like this? it could be installed by default in venv for every python plugin. @xor-xor what do you think?